Product Designer/assets/css/styles.css

```diff
@@ -0,0 +1,203 @@
+/* UNIVERSAL STYLES */
+*{
+    font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+    color: white;
+    padding: 0;
+    box-sizing: border-box;
+    margin: 0px;
+}
+body{
+    background-color: black;
+}
+
+/* MAIL SECTION STYLES*/
+.mailSection{
+    width: 74%;
+    margin: 0 auto;
+    display: flex;
+    justify-content: end;
+    font-size: 2.5em;
+    font-weight: 900;
+    height: 163px;
+    align-items: center;
+}
+.mailSection h6{
+    letter-spacing: 0.5px;
+    font-size: 0.6em;
+}
+
+/* NAME SECTION STYLES */
+.nameSection{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    height: 485px;
+    width: 74%;
+    margin: 0 auto;
+}
+.name{
+    font-size: 150px;
+    margin: 0;
+}
+.name span{
+    font-weight: 300;
+    letter-spacing: 3px;
+}
+.role span{
+    color: #4b90e2;
+}
+.role{
+    font-size: 47px;
+}
+.mobileEmail{
+    display: none;
+}
+
+/* PRODUCTS SECTION  */
+.productsContainer{
+    display: flex;
+    flex-wrap: wrap;
+    width: 80%;
+    margin: 0 auto;
+    justify-content: space-between;
+    row-gap: 50px;
+}
+.productsContainer img{
+    width: 427px;
+}
+
+
+/* LOGO SECTION STYLES */
+.logoContainer{
+    width: 80%;
+    margin: 0 auto;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    flex-wrap: wrap;
+}
+.logoTitle{
+    margin-top: 120px;
+    margin-bottom: 120px;
+    font-size: 47px;
+    color: #CECECE;
+    font-weight: 700;
+}
+.logoContainer span{
+    color: #B8E985;
+}
+.logoRow{
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-around;
+    align-items: center;
+    width: 100%;
+}
+.logoContainer img{
+    width: 120px;
+}
+#adidas{
+    width: 100px;
+}
+
+/* FOOTER SECTION STYLES */
+.footer{
+    display: flex;
+    margin: 0 auto;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    height: 600px;
+    width: 80%;
+    margin-top: 100px;
+}
+.iconsSection{
+    display: flex;
+    width: 400px;
+    justify-content: space-around;
+    align-items: center;
+    height: 240px;
+}
+.iconsSection img{
+    height: 80px;
+}
+.breakLines{
+    height: 27px;
+    width: 75%;
+    display: flex;
+    justify-content: center;
+    align-items: flex-end;
+}
+.shortLine{
+    background-color: white;
+    height: 2px;
+    width: 45px;
+    margin-right: 30px;
+}
+.longLine{
+    background-color: white;
+    height: 2px;
+    width: 210px;
+}
+
+/* STYLES FOR SMALLER SCREENS */
+@media only screen and (max-width: 1163px){
```

Comment:
Smaller in sense ?, what resolutions are you targeting ?

Reply:
Sry that i should have changed it to 474px now it is corrected for mobile view ...